worker/upload_test.go

```diff
@@ -127,14 +133,250 @@ func TestUploadDownload(t *testing.T) {
 	}
 }
 
-func testParameters(bucket, path string) uploadParameters {
+func TestUploadPackedSlab(t *testing.T) {
+	// create upload params
```

Comment:
Would be nice to deduplicate the first 20 or so lines of this test. Maybe into something like a mockWorker.

Reply:
That was coming in the next one but I can add it here.